index.html

```diff
@@ -52,22 +52,31 @@ <h1 class="masthead-heading mb-0">GEOSTAC</h1>
         </div>
     </header>
     <section class="page-section  text-black mb-0 bg-secondary" id="problem">
```

Comment:
This id needs to be renamed to something different. This is causing the problems section to not reappear when you click back on the problems tab.